tests/test_circuits.py

```diff
@@ -310,15 +311,17 @@ def test_unitary_gate_u_u3(gate, theta, phi, lambd):
 
 def test_swap():
     U = X(0)
-    U += SWAP(0,2)
+    U += SWAP(0, 2)
     wfn = simulate(U)
     wfnx = simulate(X(2))
     assert numpy.isclose(numpy.abs(wfn.inner(wfnx))**2,1.0)
 
     U = X(2)
-    U += SWAP(0,2, power=2.0)
+    U += SWAP(0,2, power=3.0)
```

Comment:
I don't understand the test the way it was written before. SWAP^2 = I, so this simply expects X(2) to have the same output as X(0)? In comparison, a few lines later, a SWAP with power 2 is tested in a context where it makes sense to not actually swap.

I'm not sure, but I assumed that here we actually want to execute a swap, while testing the power argument, so I changed it to 3.